dist/yoast-seo.js

```diff
@@ -2132,13 +2139,14 @@ YoastSEO = ( "undefined" === typeof YoastSEO ) ? {} : YoastSEO;
  * @param {YoastSEO.App} args
  * @constructor
  */
-YoastSEO.ScoreFormatter = function( args ) {
-	this.scores = args.pageAnalyzer.analyzeScorer.__score;
-	this.overallScore = args.pageAnalyzer.analyzeScorer.__totalScore;
-	this.outputTarget = args.config.targets.output;
-	this.overallTarget = args.config.targets.overall;
+YoastSEO.ScoreFormatter = function( scores, overallScore, outputTarget, overallTarget, keyword, saveScores ) {
```

Comment:
This kind of defeats the purpose. Please do pass in an `args` object. Just only let it have these six properties.